nailgun/entity_mixins.py

```diff
@@ -551,17 +552,30 @@ def to_json_dict(self):
         json_dct = {}
         for field_name, field in fields.items():
             if field_name in values:
-                if isinstance(field, OneToOneField):
-                    json_dct[field_name] = values[field_name].to_json_dict()
+                value = values[field_name]
+                if value is None:
+                    json_dct[field_name] = None
+                elif isinstance(field, OneToOneField):
+                    json_dct[field_name] = value.to_json_dict()
                 elif isinstance(field, OneToManyField):
                     json_dct[field_name] = [
-                        entity.to_json_dict() for entity in values[field_name]
+                        entity.to_json_dict() for entity in value
                     ]
                 else:
-                    json_dct[field_name] = to_json_serializable(
-                        values[field_name])
+                    json_dct[field_name] = to_json_serializable(value)
```

Comment:
At first I was doing it. But discovered a case during test on robottelo where an OneToOne is None and. 'json_dct[field_name] = value.to_json_dict()' lead to error because to_json_dict() is been called on a None value.